apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v2/metadata.json

```diff
@@ -12,18 +14,20 @@
     "description" : "gov.cms.bfd:bfd-server-war",
     "url" : "https://localhost:IGNORED_FIELD/v2/fhir"
   },
-  "fhirVersion" : "4.0.0",
-  "format" : [ "application/fhir+xml", "application/fhir+json" ],
+  "fhirVersion" : "4.0.1",
+  "format" : [ "application/fhir+xml", "xml", "application/fhir+json", "json" ],
   "rest" : [ {
     "mode" : "server",
     "resource" : [ {
       "type" : "Claim",
       "profile" : "http://hl7.org/fhir/StructureDefinition/Claim",
       "interaction" : [ {
-        "code" : "read"
-      }, {
         "code" : "search-type"
+      }, {
+        "code" : "read"
       } ],
+      "searchInclude" : [ "*", "Claim:mbi" ],
```

Comment:
need to understand the *

Reply:
Doesn't explain the star yet, but here is the change where the revinclude was added to the output in 5.4.0:

[#2521](https://github.com/hapifhir/hapi-fhir/issues/2521)

The automatically generated CapabilityStatement for R4+ will now incude the list of supported revinclude values.

Reply:
From @bfausett77 :

More information on the includes and revincludes.  I think the * is ok.  "Both _include and _revinclude use the wild card "*" for the search parameter name, indicating that any search parameter of type=reference be included. Though both clients and servers need to take care not to request or return too many resources when doing this. Most notably, using iterative wildcards inclusions might lead to the retrieval of the full patient's record, or even more than that: resources are organized into an interlinked network and broad _include paths may eventually traverse all possible paths on the server. For servers, these iterative and wildcard _includes are demanding and may slow the search response time significantly."

Also worthy to note...if we don't have a resource to return, that a request is asking to be included, then it just won't return it.  No error.  According to the FHIR Documentation...or at least that is what should happen.  "If there is no reference, or no matching resource, the resource cannot be retrieved (e.g. on a different server), then the resource is omitted, and no error is returned.

Reply:
fyi @karlmdavis